### DISCUSSION TITLE
Do amoro support iceberg rest api?

### DISCUSSION THREAD

**Discussion by liurenjie1024** [apache/amoro#2309] [Q&A] [answered]
[Rest Catalog](https://iceberg.apache.org/catalog/#decoupling-using-the-rest-catalog) is iceberg's recommended way of decoupling clients from concrete implementation. I want to ask if amoro have rest server for this?

**Comment by zhoujinsong** [comment, marked as answer]
Hi,

AMS (Amoro Management Service) implements the API of Iceberg Rest Catalog. When creating a Catalog, if you select Internal Catalog and Iceberg Format, you will use the Iceberg Rest Catalog service provided by AMS. However, you can also connect to an externally implemented Iceberg Rest Catalog service by selecting External Catalog and Iceberg Format, and choosing Custom as the meta store.

**Reply by liurenjie1024** [reply]
Really cool, thanks!
